component/events.md

```diff
@@ -50,6 +50,205 @@ Components have the ability to bind to special [can.events.inserted inserted] an
         }
       }
 
+## Listening to viewModel events
+
+As in [can.Control] we can listening for events inside or outside the Component. We can use a Templated Event Handler for this. Unlike [can.Control] the value inside `{Name}` are looked up on `viewModel` first followed by `this.options` and then the `window` object.
+You can use the events for:
+* Tracking changes on [can.List] or [can.Map] like `add`, `remove`, `change`, (`replaced`)
+* Tracking changes on properties on the viewModel
+* Tracking changes on [can.Model]-Instance like `created`, `updated`, `destroyed`
+* Tracking promises on [can.Deferred] 
+
+The following example shows all the described possibilities:
```

Comment:
This is a great example.  It might be a bit too long for someone to easily digest.  Do you think all these cases are necessary?  This seems to be documenting `can.Map` and `can.List`s events instead of how the "events" object works.